service/src/main/java/bio/terra/workspace/service/workspace/model/WorkspaceConstants.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.workspace.service.workspace.model;
+
+public class WorkspaceConstants {
+  // For now, denote data source workspaces with this key/value. This may change, as product becomes
+  // more settled.
+  // "terra-" prefix means this key is used by Terra internally (as opposed to used by user).
+  public static final String DATA_SOURCE_KEY = "terra-type";
```

Comment:
Product is now calling these datasets. I think we should follow suit in the code.